test/src/test/java/org/corfudb/integration/CorfuStoreBrowserIT.java

```diff
@@ -256,7 +256,7 @@ public void browserRegistryTableTest() throws
                 tableName,
                 SampleSchema.Uuid.class,
                 SampleSchema.Uuid.class,
-                SampleSchema.Uuid.class,
+                null,
```

Comment:
is null schema allowed?

Reply:
that is an explicit ask, so I guess it has to be 🤷